src/test/java/hw3/data/GetProperties.java

```diff
@@ -0,0 +1,26 @@
+package hw3.data;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class GetProperties {
+    protected static FileInputStream fileInputStream;
+    protected static Properties PROPERTIES;
+    static {
+        try {
+            fileInputStream = new FileInputStream("src/test/resources/hw3/conf.properties");
+            PROPERTIES = new Properties();
+            PROPERTIES.load(fileInputStream);
+        } catch (IOException e) {
+            e.printStackTrace();
+        } finally {
+            if (fileInputStream != null)
+                try {
+                    fileInputStream.close();
+                } catch (IOException e) {
+                    e.printStackTrace(); } } }
```

Comment:
Ctrl+Alt+L formatting the code in the IntelliJ IDEA. It's not according to Java convention to close } in one line

Reply:
code format updated